src/Map/MapWrapper.web.js

```diff
@@ -17,12 +18,32 @@ const MapWrapper = ({
   filteredMarkers = [],
   viewCenter,
 }) => {
+  const markers = useMemo(() => {
+    if (!filteredMarkers) {
+      return []
+    }
+
+    return filteredMarkers.map(marker => (
+      <View lat={marker.lat} lng={marker.lng} onClick={marker.onPress} key={marker.key}>
+        <Image
+          resizeMode="contain"
+          source={marker.image}
+          style={[styles.markerImage, additionalStyles.markerImage]}
+        />
+      </View>
+    ))
+  }, [filteredMarkers])
+
   return (
     <GoogleMapReact
-      bootstrapURLKeys={{ key: apiKey }}
+      apiKey={apiKey}
       defaultCenter={viewCenter}
       defaultZoom={defaultZoom}
-      options={options}
+      options={{
+        ...options,
+        mapTypeControl: false,
+        streetViewControl: false,
+      }}
```

Comment:
We could memoize this too to avoid re-renders

Reply:
Good shout, I've made this change, thanks!